src/apps/core/i18n/en-US.json

```diff
@@ -144,7 +144,10 @@
 		"defaultRules": {
 			"required": "This field is required.",
 			"remote": "Please fix this field.",
-			"email": "Please enter a valid email address.",
+			"email": {
+				"one": "Please enter a valid email address.",
```

Comment:
The e-mail rule seems to be broken when a single e-mail is validated:
![image](https://user-images.githubusercontent.com/893877/107980333-ab49b500-6f85-11eb-9db3-1a7b0022ba23.png)